tests/models/test_fluid_mass_balance.py

```diff
@@ -354,6 +352,12 @@ def test_tested_vs_testable_methods_single_phase_flow(
         ("well_fluid_flux", 0, 2),  # use dim_restriction=2 to ignore well flux
         ("well_flux_equation", 0, 2),  # use dim_restriction=2 to ignore well equation
         ("well_radius", 0.1, None),
+        # Testing of methods with deeper namespace
```

Comment:
The testing of the fluid mass balance was modified and adapted to the new interface for fluid properties.
The test was extended to cover the standard model implemented (1 phase 1 component).

Reply:
This can be resolved from my side